tests/robottelo/test_func_shared.py

```diff
@@ -154,142 +154,170 @@ def simple_shared_counter_with_exception_not_restored(index=0):
     raise NotRestorableException('error', "I'am not restorable")
 
 
-class FunctionSharedTestCase(TestCase):
-    @classmethod
-    def initiate_namespace_scope(cls):
-        # each_time generate a new name space
+class TestFuncShared:
+    @pytest.fixture(scope="class")
```

Comment:
Please switch these to single-quote strings.
It looks like there are multiple instances of this throughout the file as well.